CMakeLists.txt

```diff
@@ -7,7 +7,16 @@ set(Required_Gpgmepp_Version 1.13.1)
 
 cmake_minimum_required(VERSION ${Required_CMake_Version})
 
-PROJECT(ledger)
+if (NOT SKBUILD)
+  set(SKBUILD_PROJECT_NAME ledger)
+  set(SKBUILD_PROJECT_VERSION 3.3.2)
+endif()
```

Comment:
The `SKBUILD` variables are set when building using the [scikit-build-core](http://scikit-build-core.readthedocs.io) build backend with pip